lib/explorer/data_frame.ex

```diff
@@ -107,11 +107,18 @@ defmodule Explorer.DataFrame do
       # path to a file in disk
       Explorer.DataFrame.from_parquet("/path/to/file.parquet")
 
-      # path to a URL schema (with optional configuration) (coming soon)
-      Explorer.DataFrame.from_parquet("s3://bucket/file.parquet", config: FSS.S3.config(credentials))
+      # path to a URL schema (with optional configuration)
+      Explorer.DataFrame.from_parquet("s3://bucket/file.parquet", config: FSS.S3.Entry.config_from_system_env())
```

Comment:
Maybe something got lost in translation but this should be this (functions should be in FSS.S3):

```suggestion
      Explorer.DataFrame.from_parquet("s3://bucket/file.parquet", config: FSS.S3.config_from_system_env())
```

But in practice, `config` is optional.